### ISSUE TITLE
Starting ara shows "SyntaxError: invalid syntax" in Ubuntu Xenial

### ISSUE THREAD

**Issue by robertdebock** [ansible-community/ara#147]
## What component is this about ?

ara-manage

## What is your ARA installation like ?

Both Ansible and ARA are installed using `pip`

```
$ pip list
Package             Version
------------------- ----------------------
ara                 1.4.2
...
```

```
$ ansible --version
ansible 2.9.10
```

## What is happening ?

When starting the service, an error is displayed:

```
# /usr/local/bin/ara-manage runserver
Traceback (most recent call last):
  File "/usr/local/bin/ara-manage", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/ara/server/__main__.py", line 41, in main
    if not os.path.exists(settings.ARA_SETTINGS):
  File "/usr/local/lib/python3.5/dist-packages/django/conf/__init__.py", line 79, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python3.5/dist-packages/django/conf/__init__.py", line 66, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python3.5/dist-packages/django/conf/__init__.py", line 157, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/usr/local/lib/python3.5/dist-packages/ara/server/settings.py", line 24, in <module>
    from dynaconf import LazySettings
  File "/usr/local/lib/python3.5/dist-packages/dynaconf/__init__.py", line 6, in <module>
    from dynaconf.base import LazySettings  # noqa
  File "/usr/local/lib/python3.5/dist-packages/dynaconf/base.py", line 130
    if f"{key.upper()}_FOR_DYNACONF" in for_dynaconf_keys
                                   ^
SyntaxError: invalid syntax
```

## What should be happening ?

I was desperately hoping the service would start.

By the way, I spotted [ARA](https://www.icloud.com/photos/#0Pfk1xswYoSoOI2YJDr8-Vahw) near my house!

**Comment by dmsimard** [comment]
Nice boat! There's a tiger in my office :)

![IMG_20200702_124122](https://user-images.githubusercontent.com/1291204/86470741-0df98380-bd0a-11ea-92b3-c073c50b3320.jpg)

According to what I can see this is a trace from dynaconf trying to format a f-string but on python3.5.
f-strings started being supported after python3.6.

It looks like that change landed in dynaconf 3.0.0: https://github.com/rochacbruno/dynaconf/commit/44766bacaae3a2fc15342dd42379bd6aea356f7c

We dropped f-strings in ara in order to support 3.5 a while longer so this is unfortunate but we could probably have a python-version dependant pin like this:

```
dynaconf<3.0.0;python_version<'3.6'
dynaconf;python_version>='3.6'
```

In the meantime, could you check if the previous version of dynaconf works for you ?

By the way, which Linux distribution is this ?

Thanks,

**Comment by robertdebock** [comment]
Seems to be solved with your the release of ara.

Thanks @dmsimard!
